.github/workflows/testnode.bash

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# The script starts up the test node (with timeout 1 minute), with option to
+# run l2 transactions to make sure node is working
+
+timeout 60 ${{ github.workspace }}/nitro-testnode/test-node.bash --init --dev || exit_status=$?
```

Comment:
this doesn't use runl2Txs
Also - better if we can have all relevant logic here.
you can:
* add --detach when first callinf test-node
* wait for a little with sleep
* call e.g. test-node.sh script  send-l2 --ethamount 100 --to user_l2user --wait

Reply:
Done.